artifactory/commands/transferfiles/transfer_test.go

```diff
@@ -0,0 +1,162 @@
+package transferfiles
+
+import (
+	"encoding/json"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/transfer"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"sync"
+	"testing"
+	"time"
+)
+
+const uuidTokenForTest = "af14706e-e0c1-4b7d-8791-6a18bd1fd339"
+
+type validateVersionTestSuite struct {
+	testName      string
+	curVersion    string
+	errorExpected bool
+}
+
+func TestValidateDataTransferPluginMinimumVersion(t *testing.T) {
+	var pluginVersion string
+	testServer, serverDetails, _ := transfer.CreateRestsMockServer(t, func(w http.ResponseWriter, r *http.Request) {
+		if r.RequestURI == "/"+pluginsExecuteRestApi+"dataTransferVersion" {
+			content, err := json.Marshal(utils.VersionResponse{Version: pluginVersion})
+			assert.NoError(t, err)
+			_, err = w.Write(content)
+			assert.NoError(t, err)
+		}
+	})
+	defer testServer.Close()
+
+	srcPluginManager, err := createSrcRtUserPluginServiceManager(serverDetails)
+	if err != nil {
+		assert.NoError(t, err)
+		return
+	}
+
+	testsArray := []validateVersionTestSuite{
```

Comment:
I think the convention in Table Testing is to put the test cases outside the test method. This way it makes the method shorter and more readable.
Let's make this change in the other tests too.